Package.swift

```diff
@@ -322,6 +322,7 @@ let package = Package(
                 "TuistGraph",
                 "TuistSupport",
                 rxSwiftDependency,
+                "CryptoSwift",
```

Comment:
What about using Swift's CryptoKit library with the goal of minimizing the binary size?

Reply:
It seems that are 2 libraries that could be used. There's Apple's Open Source [Swift Crypto](https://github.com/apple/swift-crypto), which can be used in any platform, and there's [CryptoKit](https://developer.apple.com/documentation/cryptokit). This will require users to be on macOS 10.15+.

I don't see an advantage regarding binary size on replacing CryptoSwift with SwiftCrypto, besides the fact that one is "official" by Apple and will probably be maintained longer.

Reply:
I've implemented with CryptoKit in the end.